tests/integration/components/uni-carrousel-item-test.js

```diff
@@ -0,0 +1,24 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('uni-carrousel-item', 'Integration | Component | uni carrousel item', {
+  integration: true
```

Comment:
Please do some tests. CircleCI is not working for this but we still need to keep the tests passing :)